### PR TITLE
Better english

### PR DESCRIPTION
**Pull request by HyxonQz**
Improved the markdown, grammar and english quality of several files. Most importantly, `readme.md` has been improved to make more sense. In turn, this should make understanding the aim and potential of the project easier for non native english speakers.

### CODE REVIEW

File: src/common/application-event.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export interface ApplicationEvents {
    */
   instance: (event: InstanceEvent) => void
   /**
-   * Broadcast instance to inform other applications nodes in cluster about its existence
+   * Broadcast instance to inform other application nodes in cluster about it's existence
```
Comment (aidn3) [inline src/common/application-event.ts]: `applications` as other nodes connecting via websocket. There can be multiple ones.
I believe it is `its`

File: README.md [hunk 2]
```diff
@@ -5,26 +5,28 @@
...
-- Project have moved to ESM. Old plugins that still use commonjs will not work anymore
-- Minecraft bot and websocket have been stripped down to bare metal. Any plugin that use those features must update their code.
-- Punishment system has been revised and now uses `./configs/` to save its state as a file. Directory is now git-ignored.
-- any unexpected error during the launching phase will result in a shutdown with an exit code instead of sending a notification
+- `config.yaml` has been reformatted. Check the new `config_example.yaml` and compare it with the old config file.
+- Version 3.x websocket is incompatible with older versions. You cannot connect the new version with the old one.
+- Chat Commands have been moved to their own instance in `config.yaml`. Only enable one instance when using websocket.
+- Project has been moved to ESM. Old plugins that still use commonjs will not work anymore.
+- The Minecraft bot and websocket have been stripped down to bare metal. Any plugins that use those features must be updated accordingly.
+- The punishment system has been revised and now uses `./configs/` to save it's state as a file. The `./configs/` directory is now git-ignored.
```
Comment (aidn3) [inline README.md]: `its` too I believe.

File: README.md [hunk 3]
```diff
@@ -55,103 +57,103 @@ Breaking changes:
...
-- Supports Proxies for Minecraft instances
+- Supports proxies for Minecraft instances
 
 ### Permissions
 
 | Name    | Description                                                                                                        |
 | ------- | ------------------------------------------------------------------------------------------------------------------ |
-| Anyone  | Anything that usually can be seen in-game guild chat<br> e.g. Public Chat, Join/Leave Notification, etc            |
-| Helper  | Anything that requires special permission to see in-game guild chat<br> e.g. Officer Chat, Mute notification, etc. |
+| Anyone  | Anything that usually can be seen in-game guild chat<br> e.g. public chat, join/leave motification, etc            |
```
Comment (aidn3) [inline README.md:75]: notification*

File: README.md [hunk 4]
```diff
@@ -55,103 +57,103 @@ Breaking changes:
...
+| punishment-plugin | Enforces punishments and prevents punishment evasion                                    | Recommended     | Plugin will mute users but won't ban them, instead it will send a notification in officer chat.                                       |
+| reaction-plugin   | Send a greeting/reaction message when a member joins/leaves or is kicked from the guild | Quality of Life | N/A                                                                                                                                   |
+| dark-auction      | Send a reminder when a skyblock dark auction is starting                                | Quality of Life | Sends a reminder 5 minutes and 1 minute before the dark auction.                                                                      |
+| starfall-cult     | Send a reminder when the skyblock starfall cult gathers                                 | Quality of Life | N/A                                                                                                                                   |
 
-### How to execute commands
+### How to execute commands (override)
```
Comment (aidn3) [inline README.md:154]: Changing title will break table of contents links. Need to fix them.

File: src/common/application-event.ts (hunk 1 above)
Comment (HyxonQz) [reply src/common/application-event.ts]: Changed to "bridge", fixed instances of "it's" being contracted, not possessive

File: README.md (hunk 2 above)
Comment (HyxonQz) [reply README.md]: Fixed instances of "it's" being contracted, not possessive

File: README.md (hunk 3 above)
Comment (HyxonQz) [reply README.md]: Typo fixed

**Review by aidn3 — APPROVED** [review]

**Review by aidn3 — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (HyxonQz) [reply README.md:154]: I believe ToC should be updated then, title is ambiguous as it only covers one command (`override`) but the title infers all commands

Comment (aidn3) [reply README.md:154]: Then updated it :)

Comment (HyxonQz) [reply README.md]: See commit `1cbeeb9a2a4b6cd9c3d05c0d52c4da14e61eb2e5`

**Review by aidn3 — APPROVED** [review]

**Review by aidn3 — APPROVED** [review]

**Pull request merged**